docs/dietpi_tools/system_maintenance.md

```diff
@@ -94,18 +106,25 @@ dietpi-backup
     - `.dietpi-backup_stats` gives a list of completed operations with time and date
     - `.dietpi-backup.log` gives a list of every processed file
 
-### Location of the backup
+### Important DietPi-Backup hints
 
-The location of the backup can be set to any directory which is not part of the backup itself. Especially, these options are typical:
+!!! note "File system type of the backup directory: NFS needed!"
```

Comment:
It should be made clear that this applies for network drives only, actually not even there, as SSHFS would work as well. Probably better:
```suggestion
!!! note "File system type of the backup directory: Linux filesystem needed!"
```
or the other way round, that Samba is not supported?